test/test_crypto.py

```diff
@@ -275,6 +275,18 @@ def test_verify_cert(self):
         except CryptoException:
             pass
 
+    def test_message_tampering(self):
+        """Test that a tampered message is not accepted as valid."""
+        signed_message = sign(MSG, TEST_CERT_FILE, TEST_KEY_FILE)
+        tampered_message = signed_message.replace(MSG, "Spam")
+
+        # Verifying the orignal, un-tampered message should be fine.
+        verify(signed_message, TEST_CA_DIR, False)
```

Comment:
Should either do a check on the output of `verify` or remove this line - it's not really doing anything in a test case.

Reply:
Yes we should. I have added a check on both the message returned and the DN of the signer.